### ISSUE TITLE
Femoral nerve distribution to findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10805]
**Preferred term label:**
Femoral nerve distribution to findings

**Synonyms**

**Definition (free text, please give PubMed ID)**
Paresthesias in anterior/medial aspect of the thigh (anterior and medial cutaneous nerves of the thigh) and/or medial aspect of the lower leg and foot (saphenous distribution). Weakness in the quadriceps.

**Parent term (use hpo.jax.org/app)**
Parent https://hpo.jax.org/browse/term/HP:0012181, the femoral term would be similar to the finding for another nerve: https://hpo.jax.org/browse/term/HP:0012185

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hip dislocation, acquired
Heterotopic ossification (HO), nongenetic
(more diseases will be listed when our orthopedist gets to the lower extremity) 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001136
- label: Femoral nerve distribution
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Paresthesias in anterior/medial aspect of the thigh (anterior and medial cutaneous nerves of the thigh) and/or medial aspect of the lower leg and foot (saphenous distribution). May be accompanied by weakness in the quadriceps.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10805
- synonyms: Femoral nerve distribution(NONE)
